bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -206,7 +206,7 @@ spec:
                 - --enable-leader-election
                 command:
                 - /manager
-                image: quay.io/opentelemetry/opentelemetry-operator:v0.28.0
+                image: controller:v0.28.0
```

Comment:
No, sorry, I meant that this should be the actual image. We use a tool called "kustomize", which will take templates (bases) and merge with customizatoins. This is all within the config directory. The bundle is not a template nor a customization, it's a final result, similar to the manifest we generate during the release workflow. Therefore, whatever is inside the config directory should use "controller" for the operator image name, but things outside of it should use an actual image.